src/docking/search_nci.f90

```diff
@@ -199,7 +199,7 @@ END SUBROUTINE Quicksort
       write (env%unit, *) '|         Starting Energy Screening          |'
       write (env%unit, *) '=============================================='
       write (env%unit, *)
-      if (.not. fulle) write (env%unit, *) ' Fast Mode selected (recommended)'
+      if (.not. fulle) write (env%unit, *) ' No ATM term employed (recommended)'
       if (.not. fulle) write (env%unit, *) ' If ATM term should be included, use -atm option.'
```

Comment:
Is it `-atm` or `--atm`? Or does both work? The docs say two dashes.

Reply:
Both works as for most of the `xtb` flags.